main.js

```diff
@@ -2,6 +2,8 @@
 
 const { app } = require("electron");
 const { start, getLastWindow } = require("./index");
+const client = require('./src/client.js')
+const consts = require('./src/consts.js')
```

Comment:
I know this is only a draft, but you should be able to get `client` and `consts` like this `const { start, getLastWindow, client, consts } = require("./index");` if you export them at the bottom of `index.js`